content/install/amazon.md

```diff
@@ -137,6 +139,30 @@ A string containing the identifier of the subnet used to configure instance netw
 DRONE_AMAZON_SUBNET_ID=subnet-0b32177f
 ```
 
+### DRONE_AMAZON_IMAGE
+
+A string field, provides the image (AMI) you wish to use when creating instances.
+
+```
+DRONE_AMAZON_IMAGE=ami-66506c1c
```

Comment:
this should be listed as optional and should specify that it must be an ubuntu image.  The default cloud-init file would not work with other distros and would fail.